monitoring/testonly/delta.go

```diff
@@ -15,45 +15,34 @@
 package testonly
 
 import (
-	"fmt"
-	"strings"
-
 	"github.com/google/trillian/monitoring"
 )
 
-// CounterSnapshot records the value of a counter at a specific point in time.
+// CounterSnapshot records the latest value from a time series in a counter.
+// This value can then be compared with future values. Note that a counter can
+// contain many time series, but a CounterSnapshot will track only one.
+// A CounterSnapshot is useful in tests because counters do not reset between
```

Comment:
Is it safe to assume that tests don't run concurrently? That would make counter checks unreliable.

Reply:
I've added some comments to state that tests using CounterSnapshot should not be run in parallel. Someone would have to add a call to `t.Parallel()` to the tests in order for this to be a problem.